src/components/addcar/AddCar.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { baseURL } from '../../helpers/api';
+import InputChange from './hooks/InputChange';
+import UseFileUpload from './hooks/useFileUpload';
+import Container from './resusable/container/Container';
+import FileUpload from './resusable/inputFields/FileUpload';
+import Input from './resusable/inputFields/Input';
+import isUserSigned from '../../helpers/auth';
+
+const AddCar = () => {
```

Comment:
- [ ] In the mobile implementation, the "add car" section form fields are a little to compressed, and need more styling:

![image](https://user-images.githubusercontent.com/57421823/223582248-536c98fc-9e1b-4d0a-823a-b54a127632e7.png)

Kindly, add the required styling for this section so the UX is improved.  💪